web/tailwind.config.js

```diff
@@ -4,7 +4,14 @@ module.exports = {
   content: ['./src/**/*.{js,ts,jsx,tsx}'],
   theme: {
     container: {
-      center: true
+      center: true,
+      screens: {
+        sm: '600px',
+        md: '728px',
+        lg: '984px',
+        xl: '984px',
+        '2xl': '984px',
```

Comment:
IIUC. This changes the default `max-width` values for the `container` class. Do we need to do this? Could we instead use a page or section-level component that sets a `max-width`?